packages/server/lib/project-base.ts

```diff
@@ -234,7 +234,7 @@ export class ProjectBase<TServer extends Server> extends EE {
     await this.saveState(stateToSave)
 
     await Promise.all([
-      checkSupportFile({ configFile: cfg.configFile, supportFile: cfg.supportFile }),
+      checkSupportFile({ supportFile: cfg.supportFile }),
```

Comment:
Do we need `Promise.all` now it's just one value?

Reply:
Done in [84bfdcc](https://github.com/cypress-io/cypress/pull/20643/commits/84bfdcc97c31c9acd6077486f9976bb6b18ddf73)